package.json

```diff
@@ -31,12 +38,13 @@
     "ethereum-waffle": "^3.4.0",
     "ethers": "^5.3.1",
     "hardhat": "^2.4.1",
+    "hardhat-abi-exporter": "^2.10.0",
     "hardhat-gas-reporter": "^1.0.4",
-    "hardhat-typechain": "^0.3.5",
+    "@typechain/hardhat": "^6.0.0",
+    "shx": "^0.3.4",
     "solidity-coverage": "^0.7.17",
     "ts-node": "^10.0.0",
-    "typescript": "^4.3.2",
-    "@openzeppelin/contracts": "^4.3.2"
+    "typescript": "^4.3.2"
```

Comment:
im honestly unsure why these package additions resulted in 22k changes coming from the `package-lock.json`